colorpickerview/src/main/java/com/skydoves/colorpickerview/ColorPickerView.java

```diff
@@ -116,6 +116,26 @@ public ColorPickerView(Context context, AttributeSet attrs, int defStyleAttr) {
     onCreate();
   }
 
+  @Override
+  public boolean isEnabled() {
+    return super.isEnabled();
+  }
```

Comment:
There is no implementation in the override function.
We can remove this override.